api/routers/edr.py

```diff
@@ -89,13 +97,24 @@ async def get_data_location_id(
 )
 async def get_data_position(
     coords: str = Query(..., example="POINT(5.179705 52.0988218)"),
-    parameter_name: str = Query(..., alias="parameter-name", example="dd,ff,rh,pp,tn"),
-    datetime: str | None = None,
+    parameter_name: Annotated[str | None, Query(alias="parameter-name", example="dd,ff,rh,pp,tn")] = None,
+    datetime: Annotated[str | None, Query(alias="datetime", example="2022-12-31T00:00Z/2023-01-01T00:00Z")] = None,
     f: str = Query(default="covjson", alias="f", description="Specify return format."),
```

Comment:
You only need to give `alias` if the HTTP name is different from the Python name. Because `parameter-name` is not a valid Python name...